### ISSUE TITLE
Photodiode calibration

### ISSUE THREAD

**Issue by nbonacchi** [int-brain-lab/iblrig#356]
Thanks for that information! I was able to switch over to the development branch and start pybpod. 

Following the instructions for setting up the rig, the next step was to calibrate the frame2TTL. I ran this protocol and it wasn't able to recommend a threshold. This is an improvement over my previous attempts where nothing was output, so it seems moving to the development branch helped with that. The rig setup manual says to check the IR filter on the frame2TTL sensor, which I will do. However, should I see anything on the screen during this calibration? I don't see any change in the screen near the sensor. I recognize this question is probably outside the scope of my original question, so feel free to push this to a different issue. I looked through the old issues and couldn't find anything on this point, but if I missed it, I'm sorry. Please direct me to a good resource to answer this question.

Thanks!

~Nick

_Originally posted by @nmtimme in https://github.com/int-brain-lab/iblrig/issues/355#issuecomment-843409895_

**Comment by nbonacchi** [comment]
When calibrating the photodiode sensor the lighting conditions are super important...
The calibration protocol will open a gray screen with a white and black square in the bottom right of the iPad screen to measure for both light intensities and try to find good values for the Schmitt trigger.
Please make sure the Bonsai workflow that controls the visual stimulus indeed does launch and the little synch square patch is indeed displayed on the iPad screen. You can do this by removing the photodiode and looking at the bottom right corner of the screen.
Then make sure the photodiode is centered with the synch patch. and make sure external light does not enter by closing the sound attenuation box. Alternatively, if you don't have a sound attenuation box you can try covering the sensor with a black cloth for the calibration to be able to give you some values. 
The calibration of these values should be in an environment as close as possible to the task execution environment. So if you cover the sensor with a cloth and the sun the task without the cloth the signals that the sensor will give you might not be correct.

**Comment by nmtimme** [comment]
Hi @nbonacchi ,

The small box in the lower right corner does not appear. The ipad screen is on and I believe I set it up properly as the other monitor. This is the output from pybpod:

![image](https://user-images.githubusercontent.com/38138179/118829705-5707e700-b88c-11eb-809d-014c08df01d1.png)

Indeed it appears that something is wrong with Bonsai. I could not find Bonsai version 2.5.1 (as discussed in the rig setup manual) on the Bonsai website and instead I installed the most recent version (2.6.2, I believe). If version 2.5.1 is necessary, where should I obtain it? If it is some other problem, I'd appreciate any suggestions you might have. Thank you so much for all your help!

~Nick
